src/aleph/vm/orchestrator/metrics.py

```diff
@@ -81,6 +94,16 @@ async def save_record(record: ExecutionRecord):
         session.close()
 
 
+async def delete_record(execution_uuid: str):
+    """Delete the resource usage in database"""
+    session = Session()  # undefined name 'Session'
```

Comment:
Why not use an [AsyncSession](https://docs.sqlalchemy.org/en/20/orm/extensions/asyncio.html#sqlalchemy.ext.asyncio.AsyncSession) ?

Reply:
I have followed the same pattern than the rest of the methods. Maybe can you add me here an example of how to use it?

Reply:
I think it is like a regular Session but with `await` everywhere?

Reply:
I guess we reverted to `Session` then?